samples/highcharts/demo/flame/demo.js

```diff
@@ -0,0 +1,1205 @@
+        "value": 397316,
+        "color": colors[0],
+        "x": -7,
+        "low": 0,
+        "high": 397316
+    }, {
+        "name": "getArgs",
+        "id": "5",
+        "value": 397316,
+        "color": colors[0],
+        "x": -6,
+        "low": 0,
+        "high": 397316
+    }, {
+        "name": "a.Chart",
+        "id": "4",
+        "value": 397316,
+        "color": colors[0],
+        "x": -5,
+        "low": 0,
+        "high": 397316
+    }, {
+        "name": "a.chart",
+        "id": "3",
+        "value": 397316,
+        "color": colors[0],
+        "x": -4,
+        "low": 0,
+        "high": 397316
+    }, {
+        "name": "~",
+        "id": "2",
+        "value": 397316,
+        "color": colors[0],
+        "x": -3,
+        "low": 0,
+        "high": 397316
+    }, {
+        "name": "(root)",
+        "id": "1",
+        "value": 397316,
+        "color": colors[2],
+        "x": -2,
+        "low": 0,
+        "high": 397316
+    }];
+    
+(function(H){
+    H.seriesType('flame', 'columnrange', {
+        cursor: 'pointer',
+        dataLabels: {
+            enabled: true,
+            format: '{point.name}',
+            inside: true,
+            align: 'center',
+            crop: true,
+            overflow: 'none',
+            color: 'black',
+            style: {
+                textOutline: 'none',
+                fontWeight: 'normal'
+            }
+        },
+        point: {
+            events: {
+                click: function() {
+                    var point = this,
+                        chart = point.series.chart,
+                        series = point.series,
+                        xAxis = series.xAxis,
+                        yAxis = series.yAxis;
+
+                    xAxis.setExtremes(xAxis.min, point.x, false);
+                    yAxis.setExtremes(point.low, point.high, false);
+                    
+                    chart.showResetZoom();
+                    chart.redraw();
+                }
+            }
+        },
+        pointPadding: 0,
+        groupPadding: 0,
+    }, {
+            drawDataLabels: H.seriesTypes.line.prototype.drawDataLabels
+    });
+}(Highcharts))
+
+Highcharts.chart('container', {
+    chart: {
+        inverted: true,
+        marginTop: 100
+    },
+    title: {
+        text: 'Flame chart (layout: flame)',
+        y: 70
+    },
+    subtitle: {
+        text: 'Highcharts chart rendering process',
+        y: 85
+    },
+    legend: {
+            enabled: false
+    },
+    xAxis: [{
+        visible: false
+    }, {
+        visible: false,
+        startOnTick: false,
+        endOnTick: false
+    }],
+    yAxis: [{
+            visible: false
+    }, {
+        visible: false,
+        min: 0,
+        maxPadding: 0,
+        startOnTick: false,
+        endOnTick: false
+    }],
+    series: [{
+        type: 'flame',
+        data: data,
+        yAxis: 1,
+        xAxis: 1
+    }, {
+            visible: false,
```

Comment:
Indenting is off several places in this file.

Reply:
Fixed.